client/client_test.go

```diff
@@ -314,7 +314,7 @@ func TestSubscribe(t *testing.T) {
 	// Make sure the event was not received.
 	select {
 	case <-time.After(time.Millisecond):
-	case err = <-errChan:
+	case <-errChan:
```

Comment:
👍 Ther is much more of this through out the code base. It is probably worth a follow-up PR just to get rid of all the lint warnings.